Issue with the webpack blocks exports

Hi, I'm trying to set up a project using the release 1.0 branch, and I encountered some issues I wasn't able to resolve, here is the error stack. 
```
module.js:487
    throw err;
    ^

Error: Cannot find module 'webpack/lib/Chunk'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/mf/Code/accurapp/packages/webpack-preset-accurapp/node_modules/@webpack-blocks/extract-text/node_modules/extract-text-webpack-plugin/index.js:9:13)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
```
I think it's an issue with the `extract-text-webpack-plugin` version, and probably related to #190, but I'm not using the `extract-text` package in my config.

Anyway, if you want to check out the config I'm working on [here it is](https://github.com/accurat/accurapp/blob/webpack-updates/packages/webpack-preset-accurapp/index.js), to replicate that issue I just ran `node index.js`.

Instead, when testing the config in the complete build process, this error pops up:

```
TypeError: Cannot read property 'DefinePlugin' of undefined
    at accuPreset (/Users/mf/Code/accurapp/test-app/node_modules/webpack-preset-accurapp/index.js:67:18)
    at Object.<anonymous> (/Users/accurat_mf/Code/accurapp/test-app/webpack.config.js:2:18)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at readWebpackConfig (/Users/accurat_mf/Code/accurapp/test-app/node_modules/accurapp-scripts/scripts/_utils.js:60:10)
    at Object.<anonymous> (/Users/accurat_mf/Code/accurapp/test-app/node_modules/accurapp-scripts/scripts/build.js:13:16)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
```
It's telling me the variable `webpack` I require from webpack-blocks is undefined.

Feel free to ask any question about our usage of webpack blocks, might be useful some feedback, to test the configuration there is a `yarn create-test-app` command in the root.

--- a/README.md
+++ b/README.md
@@ -42,10 +42,10 @@ yarn add --dev webpack webpack-blocks
 The following sample shows how to create a webpack config with Babel support, dev server and Autoprefixer.
 
 ```js
+const webpack = require('webpack')
 const {
   createConfig,
   match,
-  webpack,
 
   // Feature blocks
   babel,

--- a/packages/webpack-blocks/README.md
+++ b/packages/webpack-blocks/README.md
@@ -14,6 +14,7 @@ Here is a small sample configuration. Instead of requiring from `@webpack-blocks
 Of course you can still separately define or install custom blocks and use them as you want.
 
 ```js
+const webpack = require('webpack')
 const {
   addPlugins,
   babel,
@@ -25,8 +26,7 @@ const {
   extractText,
   match,
   setOutput,
-  uglify,
-  webpack
+  uglify
 } = require('webpack-blocks')
 
 module.exports = createConfig([

--- a/packages/webpack/README.md
+++ b/packages/webpack/README.md
@@ -11,7 +11,7 @@ This is the `webpack` block providing webpack core functionality. Also provides 
 
 ```js
 const HtmlWebpackPlugin = require('html-webpack-plugin')
-const { addPlugins, createConfig, entryPoint, env, setOutput, sourceMaps } = require('@webpack-blocks/webpack')
+const { addPlugins, createConfig, defineConstants, entryPoint, env, setOutput, sourceMaps } = require('@webpack-blocks/webpack')
 const { css } = require('@webpack-blocks/assets')
 
 module.exports = createConfig([
@@ -25,7 +25,7 @@ module.exports = createConfig([
     })
   ]),
   defineConstants({
-    'process.env': process.env || 'development'
+    'process.env.NODE_ENV': process.env.NODE_ENV || 'development'
   }),
   env('development', [
     // will only enable source maps if `NODE_ENV === 'development'`

--- a/packages/webpack/README.md
+++ b/packages/webpack/README.md
@@ -11,7 +11,7 @@ This is the `webpack` block providing webpack core functionality. Also provides 
 
 ```js
 const HtmlWebpackPlugin = require('html-webpack-plugin')
-const { addPlugins, createConfig, entryPoint, env, setOutput, sourceMaps, webpack } = require('@webpack-blocks/webpack')
+const { addPlugins, createConfig, entryPoint, env, setOutput, sourceMaps } = require('@webpack-blocks/webpack')
 const { css } = require('@webpack-blocks/assets')
 
 module.exports = createConfig([
@@ -22,11 +22,11 @@ module.exports = createConfig([
     new HtmlWebpackPlugin({
       inject: true,
       template: './index.html'
-    }),
-    new webpack.DefinePlugin({
-      'process.env': JSON.stringify(process.env || 'development')
     })
   ]),
+  defineConstants({
+    'process.env': process.env || 'development'
+  }),
   env('development', [
     // will only enable source maps if `NODE_ENV === 'development'`
     sourceMaps()

